test/ui-test/src/screens/StatusMainScreen.py

```diff
@@ -193,3 +194,10 @@ def navigate_to_edit_profile(self):
     def close_popup(self):
         # Click in the corner of the overlay to close the popup
         click_obj_by_name_at_coordinates(MainScreenComponents.POPUP_OVERLAY.value, 1, 1)
+    
+    def is_secure_your_seed_phrase_banner_visible(self, value: bool):
+        verify(
+            squish.waitFor(lambda: is_found(MainScreenComponents.SECURE_YOUR_SEED_PHRASE_BANNER.value) is value, 15 * 1000), 
```

Comment:
Here we don't directly use `squish API`, we go through the `SquishDriver` instead to separate the automation tool from the screen's classes. If we already don't have a wrapper for the `waitFor` in the `SquishDriver`, I suggest you to add it.

Reply:
should we move 'verify' there also, because it is also squish method?

Reply:
fixed